lib/recurly/pricing/subscription/calculations.js

```diff
@@ -63,11 +63,10 @@ export default class Calculations {
     this.price.now.subtotal += this.price.now.addons;
     this.price.next.subtotal += this.price.next.addons;
 
+    this.setupFee();
```

Comment:
This was moved prior to discount calculation in order to determine whether a coupon discount amount has exceeded the discountable subtotal for the subscription cycles.